Description: Fixes MANIFEST.in so that alembic.ini is packaged
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2016-03-22

--- /dev/null	2015-11-25 10:13:06.495404578 +0000
+++ b/MANIFEST.in	2016-03-22 13:57:39.455683468 +0000
@@ -0,0 +1 @@
+recursive-include ironic *
